src/app/client/src/app/modules/core/components/content-type/content-type.component.ts

```diff
@@ -168,11 +165,7 @@ export class ContentTypeComponent implements OnInit, OnDestroy {
     };
     this.formService
       .getFormConfig(formServiceInputParams)
-      .subscribe((data: any) => {
-        this.utilService.formData = data;
```

Comment:
why is this removed

Reply:
this.utilService.formData = data; is not need i just used for checking purpose only so i removed unwanted code